loan_status.rb

```diff
@@ -0,0 +1,11 @@
+module Loan_status
+    def can_country_get_loan(gdp,state)
```

Comment:
method name should be `can_country_get_loan?` and should return true and false instead of string